### ISSUE TITLE
TensorFlow Lite for micro controllers support BatchNorm?

### ISSUE THREAD

**Issue by ohadlights** [tensorflow/tensorflow#32415]
I'm looking on this file that should list supported operations for micro controllers:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/kernels/all_ops_resolver.cc

I don't see an explicit support in batch norm layer.

Does TF Lite for micro controllers support batch normalization?
generally, does it support basic architectures like resnet and mobilenet v2?

**Comment by ymodak** [comment]
For mobilenet v2 TF Lite support, this thread can be useful https://github.com/tensorflow/tensorflow/issues/28161

**Comment by rockyrhodes** [comment]
The TFLite conversion tools support folding the batch norm ops into the the convolutions.  The resulting tflite models are supported on tflite for micros.  The micro_vision example in the code base utilizes a mobilenet-derived architecture, for example.
